[2023-10-25T11:03:36.484+0530] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_dag.inaccurate scheduled__2022-12-01T00:00:00+00:00 [queued]>
[2023-10-25T11:03:36.490+0530] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_dag.inaccurate scheduled__2022-12-01T00:00:00+00:00 [queued]>
[2023-10-25T11:03:36.490+0530] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-10-25T11:03:36.498+0530] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): inaccurate> on 2022-12-01 00:00:00+00:00
[2023-10-25T11:03:36.502+0530] {standard_task_runner.py:57} INFO - Started process 32087 to run task
[2023-10-25T11:03:36.504+0530] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'first_dag', 'inaccurate', 'scheduled__2022-12-01T00:00:00+00:00', '--job-id', '198', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpkwiqvru6']
[2023-10-25T11:03:36.505+0530] {standard_task_runner.py:85} INFO - Job 198: Subtask inaccurate
[2023-10-25T11:03:36.526+0530] {task_command.py:415} INFO - Running <TaskInstance: first_dag.inaccurate scheduled__2022-12-01T00:00:00+00:00 [running]> on host harika-Latitude-5511
[2023-10-25T11:03:36.564+0530] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='first_dag' AIRFLOW_CTX_TASK_ID='inaccurate' AIRFLOW_CTX_EXECUTION_DATE='2022-12-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-12-01T00:00:00+00:00'
[2023-10-25T11:03:36.565+0530] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-10-25T11:03:36.565+0530] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', "echo 'inaccurate'"]
[2023-10-25T11:03:36.572+0530] {subprocess.py:86} INFO - Output:
[2023-10-25T11:03:36.572+0530] {subprocess.py:93} INFO - inaccurate
[2023-10-25T11:03:36.572+0530] {subprocess.py:97} INFO - Command exited with return code 0
[2023-10-25T11:03:36.584+0530] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=inaccurate, execution_date=20221201T000000, start_date=20231025T053336, end_date=20231025T053336
[2023-10-25T11:03:36.595+0530] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-25T11:03:36.603+0530] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
